src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,62 @@
+package duke.task;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import duke.DukeException;
+
+/**
+ * Represents a deadline inherits from a Task.
+ */
+public class Deadline extends Task {
+    private static String invalidDate = "Please provide date with the following format: YYYY-MM-DD";
```

Comment:
This could be made final
```suggestion
    private static final String INVALID_DATE = "Please provide date with the following format: YYYY-MM-DD";
```